spread-tests/regression/lp-1618683b/task.yaml

```diff
@@ -0,0 +1,16 @@
+summary: Check that CLONE_NEWUSER works with clone() within snap apps
+# This is blacklisted on debian because we first have to get the dpkg-vendor patches
+systems: [-debian-8]
+details: |
+    Snap-confine used to "leak" the root filesystem directory across the
+    pivot_root call. This caused checks in the kernel to fail and resulted in
+    the inability to use clone(CLONE_NEWUSER) from snaps.
+prepare: |
+    echo "Having installed the snapd-hacker-toolbelt snap"
+    snap install snapd-hacker-toolbelt
+execute: |
+    cd /
+    echo "We can clone(CLONE_NEWUSER) as a regular user and expect it to work"
+    /snap/bin/snapd-hacker-toolbelt.busybox sh -c 'clone-newuser-test'
```

Comment:
"Can you tell me what clone-newuser-test is and where does it come from?"

I addressed this in the PR: "This requires merging https://code.launchpad.net/~jdstrand/+git/snapd-hacker-toolbelt/+ref/master."

If you are wondering where that came from-- it came from me-- I wrote clone-newuser-test.c.